std/traits.d

```diff
@@ -4138,6 +4138,13 @@ template ImplicitConversionTargets(T)
     static assert(is(ImplicitConversionTargets!(string)[0] == const(char)[]));
 }
 
+unittest // bugzilla 15940
+{
+    class Foo{}
+    struct Bar{Foo foo; alias foo this;}
+    static assert(ImplicitConversionTargets!(Bar).length == 0);
```

Comment:
I don't understand, shouldn't this equal one? `Bar` can implicitly convert to `Foo`.